### ISSUE TITLE
loss keep stable af

### ISSUE THREAD

**Issue by Pichairen** [hengchuan/RDN-TensorFlow#18]

**Comment by Pichairen** [comment]
loss never go down after many batch, after many epoch loss still between 0.1 and 0.2, any help?

**Comment by hengchuan** [comment]
it may make sense to decay the learning rate after some epochs
